CHANGELOG.latest.md

```diff
@@ -1,5 +1,7 @@
+### API Changes
+* Rename revenueCatId and productId to transactionIdentifier and productIdentifier. Old values still exist but are deprecated (#211) via Toni Rico (@tonidero)
```

Comment:
Makes sense to have it in a separate section. I wonder if either it could have been tagged differently, or if we need a special label for phc. Anyways, we will always have the flexibility to manually edit the Changelog, and I think it's not possible to cover all cases automatically.

Reply:
Yeah agreed... In this case, I could have labeled it differently, but none of the tags fit exactly... So I think just having the option to update the changelog manually is enough for now.